### DISCUSSION TITLE
Question Moved to tskit forum

### DISCUSSION THREAD

**Discussion by itsmestewart** [tskit-dev/msprime#2209] [Q&A]
I have generated a tree sequence using .mut/.anc generated by Relate. 

When loaded using tskit.load, the resulting tree sequence has no population labels. 
What could be the reason for this?

Note: The original input files have population labels.
Many thank for your assistance.

Please see the attached picture 
<img width="472" alt="Screenshot 2023-10-12 at 11 15 23" src="https://github.com/tskit-dev/msprime/assets/73900141/1fa9e312-da06-4f94-986c-f01a1d9e4da9">

**Comment by petrelharp** [comment]
This may be a question for Relate - but, maybe @nspope knows the answer?

**Comment by nspope** [comment]
Yup -- looks like Relate's converter tool just doesn't construct the individual / population tables. There's no reason why it shouldn't do this (it'd only require parsing an additional input file; there's even a placeholder in the code for it). Would definitely be worth adding this feature (but that's on the Relate side of things, not tskit).

**Comment by jeromekelleher** [comment]
Cross linking with same question over in tskit: https://github.com/tskit-dev/tskit/discussions/2864
